Added the grimoire

nightbe opened the pull request:
I hope this works

nightbe committed:
Add files via upload

nightbe committed:
Delete Aine filia Saoirse - Grimoire.md.txt

nightbe committed:
Add files via upload

GuillaumeDIDIER commented:
This worked !

GuillaumeDIDIER commented:
Ok, so I'm going to be annoying but:

Spells description can just go in full in the sheet file.
Alternately, they can just go into the global grimoire. I don't think having an extra grimoire file (with every single TeFo listed even when empty) is a good idea.

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+*(Base effect, +4 Arc, +1 Conc)*
+
+
+**Name**  
```
No need to keep the template here

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+CrAq 10  
+**R:** Voice, **D:** Dia, **T:** Ind  
+Caster flics their finger and covers an individual in highly flamable oil. If it gets ignited it does +12 damage the first round, +6 damage the second round, +0 damage the third round.    
+**Mastery:** Multi    
```
Mastery are properties of how a magi knows a spell, not of the spell itself, so it doesn't belong in a file distinct from the character.

Also, I think mastery needs level in the ability, xp, and then the list of masteries options (Multi-cast, imperturbable casting etc)

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+This spell heals a single Light Wound suffered by the person touched. This spell does not heal damage from poison or disease.  
+(Base 15, +1 Touch)  
+
+**Im not done with you yet
```
Missing `**`

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+
+## Perdo Corpus
+
+**Mr Hiccup**  
```
Great name

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+The caster is enveloped in shadows making it somewhat more difficult to clearly distinguish details of the caster by sight.  
+*(Base 3)*  
+
+**Jackfrost says fuck you particularly**  
```
Couldn't we find a substitute to profanity ?

GuillaumeDIDIER commented on Magi/Aine filia Saoirse/Aine filia Saoirse - Grimoire.md:
```diff
@@ -0,0 +1,301 @@
+**Mastery:** Fast cast  
+*(Base 3, Voice +2, con +1)*
+
+**Aegis of the Hearth (20)**  
```
Those a Core book spells, we shouldn't duplicate the info, except within the character sheet itself if it includes the description of every spell.

nightbe committed:
Edited

I have added her spell list. And fixed all your comments related changes, but I will look at the general mastery later.

GuillaumeDIDIER commented:
So the Aine - Grimoire.md should be removed. Original spells go in the global grimoire, and I saw you updated Aine.md with the spells.

GuillaumeDIDIER commented:
How mastery should be formated imho
```
Mastery 4 : Acute Sense x 2, Multicast, Penetration
```

add `(xx xp)` after the level when you also keep the xp for abilities around, but that's not the case here

nightbe committed:
Changed stuff

Added the full mastery names in Aine.
Deleted the grimoire.

nightbe commented:
I will change it "one last time" and properly add the mastery in the way you showed :)

nightbe committed:
Update Aine.md

nightbe commented:
Thanks for the patience :)
